javascript/javascript1/week2/Event-application.js

```diff
@@ -0,0 +1,10 @@
+
+  let date = new Date();
+  let today = date.getDay();
```

Comment:
always use const until it doesn't let you

Reply:
ok